### PR TITLE
Improve typography

### PR DESCRIPTION
**Pull request by hghwng**
Remove ambiguous "here" in links.

### CODE REVIEW

**Comment by hghwng** [comment]
If you think such improvement is okay, I'd like to work on all the occurrences systematically.

**Comment by hghwng** [comment]
Thanks! I'd like to simplify the process by appending more commits to this PR in the following days. I'll leave a message when it's done.

**Comment by hghwng** [comment]
Hi Corey, in addition to cleaning all "here" links, I also replaced old links of crates.io to the redirected URL. Is that okay?

File: src/cargo/conventions.md [hunk 1]
```diff
@@ -29,8 +29,9 @@ To tell `cargo` to compile or run this binary as opposed to the default or other
 binaries, we just pass `cargo` the `--bin my_other_bin` flag, where `my_other_bin`
 is the name of the binary we want to work with.
 
-In addition to extra binaries, there is support for benchmarks, tests, and
-examples. The full capabilities are documented
-[here](http://doc.crates.io/book/guide/project-layout.html).
+In addition to extra binaries, `cargo` supports [more features] such as
```
Comment (marioidival) [inline src/cargo/conventions.md]: missing link to `more features`?

**Pull request merged**